src/nkr_proxy/proxy_app.py

```diff
@@ -78,8 +78,25 @@ def index_search(search_handler=None):
         raise BadRequest('Invalid search handler. Valid search handlers are: %s' % settings.INDEX_ALLOWED_APIS)
 
     user_id = request.headers.get('x-user-id', None)
-    query_string = request.query_string.decode('utf-8')
 
+    request_content_type = request.content_type
+
+    #request_headers = {}
```

Comment:
Commented (testing) code should be removed if it's not used anymore

Reply:
logger.debug lines can be added if they are in line with the rest of debug lines